network-upgrades/mainnet-upgrades/merge.md

```diff
@@ -0,0 +1,38 @@
+## Merge Upgrade Specification
+
+### Included EIPs
+Specifies changes included in the Network Upgrade.
+
+  - [x] [EIP-3675: Upgrade consensus to Proof-of-Stake](https://eips.ethereum.org/EIPS/eip-3675)
+  - [x] [EIP-4399: Supplant DIFFICULTY opcode with RANDOM](https://eips.ethereum.org/EIPS/eip-4399)
+
+### Engine API
+
+A new set of APIs is introduced as part of The Merge for the execution layer clients to communicate with the consensus layer. The specification for it is available [here](https://github.com/ethereum/execution-apis/tree/main/src/engine).
+
```

Comment:
We might include CL specs as a section.

*If* an EL upgrade requires an accompanying CL upgrade, then I think it makes sense for the specs to cross-reference eachother

Reply:
Added a link, good call!